src/lib_json/json_value.cpp

```diff
@@ -718,7 +718,7 @@ bool Value::asBool() const {
   case uintValue:
     return value_.uint_ ? true : false;
   case realValue:
-    return value_.real_ ? true : false;
+    return static_cast<bool>(value_.real_) ? true : false;
```

Comment:
the ternary here is redundant, but I guess that is how they wrote it.

Reply:
Yeah :/ I originally turned the specific warnings off for this specific file, but then decided to just fix them.